src/main/java/org/apache/commons/cli/Util.java

```diff
@@ -17,16 +17,46 @@ Licensed to the Apache Software Foundation (ASF) under one or more
 
 package org.apache.commons.cli;
 
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
 /**
  * Contains useful helper methods for classes within this package.
+ * @since 1.10.0
  */
-final class Util {
+public final class Util {
+
+    /** An array of chars that are breaks in text */
```

Comment:
I'm don't think we should duplicate what `Character.isSpaceChar(char|int)` and `Character.isWhitespace(char|int)` do.

Reply:
I am curious now 😄 Do you have link to your bug report? TY.